nncf/compression_method_api.py

```diff
@@ -136,39 +134,6 @@ def statistics(self, quickly_collected_only=False):
             stats.update(self._model.statistics(quickly_collected_only))
         return stats
 
-    def run_batchnorm_adaptation(self, config):
```

Comment:
@vanyalzr @AlexKoff88 Please take a look.

Reply:
Looks good to me. Makes more sense to abstract BN adaptation as a separate algorithm rather than keep it in CompressionAlgorithm's API.